phat/contracts/action_offchain_rollup/lib.rs

```diff
@@ -282,6 +292,39 @@ mod action_offchain_rollup {
             }
         }
 
+        ///
+        /// Get all configuration as once, it help reduce to total roundtrip when we building dApp on top of it.
+        ///
+        /// @category Configuration
+        ///
+        #[ink(message)]
+        pub fn get_configuration(&self) -> Configuration {
+            let mut _rpc = None;
+            let mut _client_addr = None;
+            if self.client.is_some() {
+                _rpc = Some(self.client.as_ref().unwrap().rpc.clone());
+                _client_addr = Some(self.client.as_ref().unwrap().client_addr);
+            }
```

Comment:
Lets avoid the unwrap:
```suggestion
            let rpc = self.client.as_ref().map(|c| c.rpc.clone());
            let client_addr = self.client.as_ref().map(|c| c.rpc.client_addr);
```